location/src/main/java/org/odk/collect/location/satellites/GpsStatusSatelliteInfoClient.kt

```diff
@@ -35,3 +43,30 @@ private class GpsStatusSatellitesLiveData(private val locationManager: LocationM
         }
     }
 }
+
+@SuppressLint("MissingPermission")
+@RequiresApi(Build.VERSION_CODES.N)
+private class GnssStatusSatellitesLiveData(private val locationManager: LocationManager) :
+    NonNullLiveData<Int>(0) {
+
+    private val gnssStatusCallback = object : GnssStatus.Callback() {
+        override fun onSatelliteStatusChanged(status: GnssStatus) {
+            super.onSatelliteStatusChanged(status)
```

Comment:
I'm guessing you don't need this super call?

Reply:
Right